Dockerfile

```diff
@@ -0,0 +1 @@
+FROM gradle:6.8.1-jdk11
```

Comment:
The project targets java 8+.  Should we set this to java 8?

Reply:
It looks like the target is actually java 9+! 
```
> Could not target platform: 'Java SE 9' using tool chain: 'JDK 8 (1.8)'.
```

Reply:
weird that the readme is out of sync but 🤷